<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Messenger</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(208, 219, 232);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 219, 232);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="loginPage">
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="usernameEdit">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit
{
font: 400 14pt &quot;Calibri&quot;;
border-radius: 10px;
border: 1px solid #dce1e6;
background: rgb(255, 255, 255);
}
QLineEdit:focus
{
border: 1px solid rgb(68, 123, 186);
}</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Username</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="loginButton">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>35</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>16</pointsize>
              <weight>87</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(68, 123, 186, 1);
border-radius: 10px;
color: rgb(255, 255, 255);
font: 700 16pt &quot;Calibri&quot;;</string>
            </property>
            <property name="text">
             <string>Log in</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>16</pointsize>
              <weight>87</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background: transparent;
font: 700 16pt &quot;Calibri&quot;;</string>
            </property>
            <property name="text">
             <string>Enter your username:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="UsersListPage">
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLineEdit" name="findUserEdit">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit
{
padding-left: 10px; 
font: 400 14pt &quot;Calibri&quot;;
border-radius: 10px;
border: 1px solid #dce1e6;
background: rgb(255, 255, 255);
}
QLineEdit:focus
{
border: 1px solid rgb(68, 123, 186);
}</string>
          </property>
          <property name="placeholderText">
           <string>Find a user...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="logoutButton2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>16</pointsize>
            <weight>87</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(68, 123, 186, 1);
border-radius: 10px;
color: rgb(255, 255, 255);
font: 700 16pt &quot;Calibri&quot;;</string>
          </property>
          <property name="text">
           <string>Log out</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QListView" name="usersListView">
          <property name="cursor" stdset="0">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QListView
{
background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 1px solid #dce1e6;
font: 400 14pt &quot;Calibri&quot;;
outline: 0
}
QListView::item
{ 
font: 400 14pt &quot;Calibri&quot;;
border-bottom: 1px solid #dce1e6; 
padding: 10px; 
}
QListView::item:hover
{ 
background-color:rgba(208, 219, 232, 100); 
}</string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="chatPage">
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="returnButton">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>16</pointsize>
            <weight>87</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(68, 123, 186, 1);
border-radius: 10px;
color: rgb(255, 255, 255);
font: 700 16pt &quot;Calibri&quot;;</string>
          </property>
          <property name="text">
           <string>‹ back</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="usernameLabel">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">background: transparent;
font: 400 16pt &quot;Calibri&quot;;</string>
          </property>
          <property name="text">
           <string>Username</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="logoutButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>16</pointsize>
            <weight>87</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(68, 123, 186, 1);
border-radius: 10px;
color: rgb(255, 255, 255);
font: 700 16pt &quot;Calibri&quot;;</string>
          </property>
          <property name="text">
           <string>Log out</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QListView" name="chatView">
          <property name="styleSheet">
           <string notr="true">QListView
{
padding: 10px;
background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 1px solid #dce1e6;
outline: 0
}
QListView::item:hover
{ 
color: rgb(0, 0, 0);
background-color: transparent;
}
QListView::item:selected
{ 
color: rgb(0, 0, 0);
background-color: transparent;
}</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLineEdit" name="messageEdit">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit
{
padding-left: 10px; 
font: 400 14pt &quot;Calibri&quot;;
border-radius: 10px;
border: 1px solid #dce1e6;
background: rgb(255, 255, 255);
}
QLineEdit:focus
{
border: 1px solid rgb(68, 123, 186);
}</string>
          </property>
          <property name="placeholderText">
           <string>Write a message...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="sendButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>16</pointsize>
            <weight>87</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(68, 123, 186, 1);
border-radius: 10px;
color: rgb(255, 255, 255);
font: 700 16pt &quot;Calibri&quot;;</string>
          </property>
          <property name="text">
           <string>Send ›</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
